contents/docs/product-analytics/autocapture.mdx

```diff
@@ -126,6 +126,39 @@ properties: {
 }
 ```
 
+## Sending custom properties with autocaptured form submissions
+
+We do not automatically capture form values with form submissions, to prevent accidental sensitive data capture. To add custom properties to form submissions, you can use the `data-ph-capture-attribute` attribute on the form element.
```

Comment:
```suggestion
To prevent accidental sensitive data capture, we do not automatically capture form values from form submissions. To add custom properties to form submissions, you can use the `data-ph-capture-attribute` attribute on the form element.
```